config/initializers/filter_parameter_logging.rb

```diff
@@ -3,4 +3,4 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += %i(passw secret token _key crypt salt certificate otp ssn)
```

Comment:
Just double-checking, this is supposed to be `passw` and not `password`?

Reply:
Yeah it uses regex so it will filter anything with that